modules/acid/src/main/ts/ephox/acid/api/colour/HsvColour.ts

```diff
@@ -18,28 +20,17 @@ const fromRgb = (rgbaColour: Rgba): Hsv => {
   if (minRGB === maxRGB) {
     v = minRGB;
 
-    return hsvColour(
-      0,
-      0,
-      v * 100
-    );
+    return hsvColour(0, 0, v * 100);
   }
 
   /* eslint no-nested-ternary:0 */
-  const d = (r === minRGB) ? g - b : ((b === minRGB) ? r - g : b - r);
-  h = (r === minRGB) ? 3 : ((b === minRGB) ? 1 : 5);
+  const d = r === minRGB ? g - b : b === minRGB ? r - g : b - r;
+  h = r === minRGB ? 3 : b === minRGB ? 1 : 5;
   h = 60 * (h - d / (maxRGB - minRGB));
   s = (maxRGB - minRGB) / maxRGB;
   v = maxRGB;
 
-  return hsvColour(
-    Math.round(h),
-    Math.round(s * 100),
-    Math.round(v * 100)
-  );
+  return hsvColour(Math.round(h), Math.round(s * 100), Math.round(v * 100));
```

Comment:
Ambivalent about this one, seems like it could get annoying. Though I assume that if the line got too long it would keep it as args on each line?

Reply:
Yes, it does put args on their own line if it gets too long